python/res/enkf/queue_config.py

```diff
@@ -37,8 +38,23 @@ class QueueConfig(BaseCClass):
     _queue_driver          = ResPrototype("driver_ref queue_config_get_queue_driver(queue_config, char*)")
     _get_num_cpu           = ResPrototype("int queue_config_get_num_cpu(queue_config)")
 
-    def __init__(self, user_config_file=None):
-        c_ptr = self._alloc(user_config_file)
+    def __init__(self, user_config_file=None, config_content=None):
+        if user_config_file is not None and config_content is not None:
+            raise IOError("Trying to create QueueConfig with multiple configurations!")
```

Comment:
I would expect both of these errors to be `ValueError`. I did also check some of the other configs, and it seems that they (myself 2 years ago) agree :smile:

Reply:
And this behavior should be tested...